src/storage/src/source/kafka.rs

```diff
@@ -185,6 +189,60 @@ impl SourceRender for KafkaSourceConnection {
         let (mut health_output, health_stream) = builder.new_output();
         let (mut stats_output, stats_stream) = builder.new_output();
 
+        let mut outputs = vec![];
+        for (
+            id,
+            SourceExport {
+                ingestion_output,
+                details,
+                storage_metadata: _,
+            },
+        ) in &config.source_exports
```

Comment:
Dropping the destructure in the loop is the same number of lines and less unwieldy 
```suggestion
        for (id, export) in &config.source_exports {
            let export = SourceExport {
                ingestion_output,
                details,
                storage_metadata: _,
            };
```